sdk/ml/azure-ai-ml/azure/ai/ml/_schema/schedule/create_job.py

```diff
@@ -124,3 +133,26 @@ def make(self, data: dict, **kwargs) -> "Job":
 class PipelineCreateJobSchema(BaseCreateJobSchema):
     type = StringTransformedEnum(allowed_values=[JobType.PIPELINE])
     settings = NestedField(PipelineJobSettingsSchema, unknown=INCLUDE)
+
+
+class CommandCreateJobSchema(BaseCreateJobSchema):
```

Comment:
is it possible to make it inheriting ParameterizedCommand? You need to guarantee that they're aligned right?

Reply:
Good point, I'll see, inheriting CommandJobSchema may be a possible approach, I didn't realize there is so many fields need to add.